.github/dependabot.yml

```diff
@@ -137,6 +137,30 @@ updates:
     schedule:
       interval: "weekly"
 
+  - package-ecosystem: "npm"
+    directory: "/packages/webhook"
+    groups:
+      slack:
+        patterns:
+          - "@slack/*"
```

Comment:
```suggestion
```

🪓 Removing this is no blocker, but we depend on just `@slack/types` for webhooks!